packages/daemon/src/formula-identifier.js

```diff
@@ -1,10 +1,11 @@
 const { quote: q } = assert;
 
 const numberlessFormulasIdentifiers = new Set([
-  'pet-store',
-  'host',
   'endo',
```

Comment:
Could add a comment that these are sorted, to maintain that invariant.

Reply:
I'll defer this to when I look into our lint configs.